stable/jenkins/values.yaml

```diff
@@ -87,6 +87,7 @@ Agent:
   # - type: Secret
   #   secretName: mysecret
   #   mountPath: /var/myapp/mysecret
+  NodeSelector: {}
```

Comment:
I see a heck of a lot of documentation repeated over and over in each helm chart. They often get out of sync from each other. Should this belong in some kind of "how to use helm charts" document instead?